client/src/pages/Text2Sql/Text2SQL.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { CContainer, CRow, CCol, CToaster } from '@coreui/react';
+import ConfigurationPanel from '../../components/ConfigurationPanel/ConfigurationPanel';
+import ChatPanel from '../../components/ChatPanel/ChatPanel';
+import ToastNotification from '../../components/ToastNotification/ToastNotification';
```

Comment:
There is a thing called relative paths in js which can be defined in jsconfig, can you look into it? 
It will be more like 
```
import ConfigurationPanel from '.@components/ConfigurationPanel/ConfigurationPanel';
```
instead of 
```
import ConfigurationPanel from '../../components/ConfigurationPanel/ConfigurationPanel';
```